FantasmoSDK/Classes/Utilities/LocationFuser.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  LocationFuser.swift
+//  FantasmoSDK
+//
+//  Created by lucas kuzma on 8/3/21.
+//
+
+import Foundation
+import CoreLocation
+
+struct LocationFuser {
+
+    var locations: [CLLocation] = []
+
+    mutating func reset() {
+        locations = []
+    }
+
+    private func calculateConfidence(_ locations: [CLLocation]) -> FMResultConfidence {
```

Comment:
we could add some more intelligence here based on the variance of the results. 2 results with low variance should be medium. 3+ results with low variance should be high.

Reply:
I am concerned that this is an ad hoc and hand-wavy departure from the mathematical foundation we got from Jan.

What exactly would low variance be? And should we do it in meters? We can get meter distance from CLLocation.

Reply:
Some thoughts on this. We know that we can get 10-15 cm accuracy on Ickstattstr, our go-to set with a ground truth. So, how about we say (after you get the meter distance from CLLocation) that:
- if we get <15cm variance across results, then the confidence is high
- If we get up to 0.5 meters variance (which is what Tier uses internally (?)), confidence is medium
- anything above it would be low.